build.proj

```diff
@@ -48,6 +49,17 @@
     <GenerateConfigurationProps Properties="@(Property)" PropertyValues="@(PropertyValue)" PropsFolder="$(BuildConfigurationFolder)" />
   </Target>
 
+  <!-- Generate launch settings support files to enable VS debugging. -->
+  <Target Name="GenerateLaunchSettingsFiles" Condition="'$(GenerateLaunchSettings)' == 'true'">
```

Comment:
Why are you doing this here as opposed to at the end of the build? Do we have a good way to keep the Test project pattern insync with what we have in test.builds?

Reply:
This solves a) the issue that the launch profile is not visible the first time VS is opened, before the test assembly is built and b) races when hooking it into the test project build.

Reply:
> Do we have a good way to keep the Test project pattern insync with what we have in test.builds?

We can share the pattern in the Directory.Build.props as both import it.

Reply:
I'm not sure sharing just TestProjectPatternPrefix is all that interesting. However I'm not sure sharing is actually necessary.

Reply:
if you are fine with it I will just undo that change and add a comment to both places to keep them in sync?

Reply:
Yes that is fine